bitnami/jaeger/templates/collector/deployment.yaml

```diff
@@ -150,37 +158,44 @@ spec:
           {{- else if .Values.collector.startupProbe.enabled }}
           startupProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.collector.startupProbe "enabled") "context" $) | nindent 12 }}
             httpGet:
-              path: /
-              port: admin
+              path: /status
+              port: healthcheck
           {{- end }}
           {{- if .Values.collector.customLivenessProbe }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.collector.customLivenessProbe "context" $) | nindent 12 }}
           {{- else if .Values.collector.livenessProbe.enabled }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.collector.livenessProbe "enabled") "context" $) | nindent 12 }}
-            tcpSocket:
-              port: admin
+            httpGet:
+              path: /status
+              port: healthcheck
           {{- end }}
           {{- if .Values.collector.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.collector.customReadinessProbe "context" $) | nindent 12 }}
           {{- else if .Values.collector.readinessProbe.enabled }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.collector.readinessProbe "enabled") "context" $) | nindent 12 }}
             httpGet:
-              path: /
-              port: admin
+              path: /status
+              port: healthcheck
           {{- end }}
           {{- end }}
           {{- if .Values.query.resources }}
           resources: {{- toYaml .Values.collector.resources | nindent 12 }}
           {{- else if ne .Values.collector.resourcesPreset "none" }}
           resources: {{- include "common.resources.preset" (dict "type" .Values.collector.resourcesPreset) | nindent 12 }}
           {{- end }}
-          {{- if .Values.collector.extraVolumeMounts }}
-          volumeMounts: {{- include "common.tplvalues.render" ( dict "value" .Values.collector.extraVolumeMounts "context" $) | nindent 12 }}
-          {{- end }}
+          volumeMounts:
+            - name: configuration
+              mountPath: /opt/bitnami/jaeger/conf
+            {{- if .Values.collector.extraVolumeMounts }}
+            {{- include "common.tplvalues.render" ( dict "value" .Values.collector.extraVolumeMounts "context" $) | nindent 12 }}
+            {{- end }}
         {{- if .Values.collector.sidecars }}
         {{- include "common.tplvalues.render" ( dict "value" .Values.collector.sidecars "context" $) | nindent 8 }}
         {{- end }}
       volumes:
+        - name: configuration
+          configMap:
+            name: {{ include "jaeger.collector.configMapName"  . }}
```

Comment:
The annotations should include the configmap checksum to ensure changes trigger a rolling update.